docs/api/solvers/cpo.rst

```diff
@@ -0,0 +1,7 @@
+CPMpy cpo interface (:mod:`cpmpy.solvers.cpo`)
```

Comment:
can you add to: https://cpmpy.readthedocs.io/en/latest/adding_solver.html

a line that says that such an RST file in this folder must be added? I didnt know...